engine/userguide/storagedriver/selectadriver.md

```diff
@@ -100,149 +213,23 @@ settings in the file, it should have the following contents:
-is used by default. Straying from this default may increase your chances of
-encountering bugs and nuances.
-- **Follow the configuration specified on the CS Engine
-[compatibility matrix](https://success.docker.com/Help/Compatibility_Matrix)**. The
- CS Engine is the commercially supported version of the Docker Engine. It's
-code-base is identical to the open source Engine, but it has a limited set of
-supported configurations. These *supported configurations* use the most stable
-and mature storage drivers. Straying from these configurations may also
-increase your chances of encountering bugs and nuances.
-
-### Experience and expertise
+### Extra steps for Docker for Mac or Docker for Windows
 
-Choose a storage driver that you and your team/organization have experience
-with. For example, if you use RHEL or one of its downstream forks, you may
-already have experience with LVM and Device Mapper. If so, you may wish to use
-the `devicemapper` driver.
-
-If you do not feel you have expertise with any of the storage drivers supported
- by Docker, and you want an easy-to-use stable Docker experience, you should
-consider using the default driver installed by your distribution's Docker
-package.
-
-### Future-proofing
-
-Many people consider OverlayFS as the future of the Docker storage driver.
-However, it is less mature, and potentially less stable than some of the more
-mature drivers such as `aufs` and `devicemapper`.  For this reason, you should
-use the OverlayFS driver with caution and expect to encounter more bugs and
-nuances than if you were using a more mature driver.
-
-The following diagram lists each storage driver and provides insight into some
-of their pros and cons. When selecting which storage driver to use, consider
-the guidance offered by the table below along with the points mentioned above.
-
-![](images/driver-pros-cons.png)
-
-### Overlay vs Overlay2
-
-OverlayFS has 2 storage drivers which both make use of the same OverlayFS
-technology but with different implementations and incompatible on disk
-storage. Since the storage is incompatible, switching between the two
-will require re-creating all image content. The `overlay` driver is the
-original implementation and the only option in Docker 1.11 and before.
-The `overlay` driver has known limitations with inode exhaustion and
-commit performance. The `overlay2` driver addresses this limitation, but
-is only compatible with Linux kernel 4.0 and later. For users on a pre-4.0
-kernel or with an existing `overlay` graph, it is recommended to stay
-on `overlay`. For users with at least a 4.0 kernel and no existing or required
-`overlay` graph data, then `overlay2` may be used.
-
-> **Note**
-> `overlay2` graph data will not interfere with `overlay` graph data. However
-> when switching to `overlay2`, the user is responsible for removing
-> `overlay` graph data to avoid storage duplication.
+Before you change the default storage driver on Docker for Mac or Docker for
```

Comment:
I have no idea. I only know that it is a possible thing. @thaJeztah any thoughts?